pkgs/by-name/gi/gitlab-ci-linter/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitLab,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "gitlab-ci-linter";
+  version = "2.4.0";
+
+  src = fetchFromGitLab {
+    owner = "orobardet";
+    repo = "gitlab-ci-linter";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-zH7eeAJzazDf4LnfBxFKMIPIB4Gx4rn7DCEqBV5zIWo=";
+  };
+
+  vendorHash = "sha256-GKv7uWnY9UqDuzj/VcXTAjGJUyUEsZws3ORnPCX8mwU=";
+
+  # Build flags based on the project's Makefile
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${finalAttrs.version}"
+  ];
+  env.CGO_ENABLED = 0;
+
+  meta = {
+    description = "gitlab-ci lint tool for validating .gitlab-ci.yml using Gitlab API. Can be used as git pre-commit hook. Can be used with custom Gitlab instance.";
```

```suggestion
    description = ".gitlab-ci.yml lint helper tool";
    longDescription = ''
      This tool uses the [Gitlab API](https://docs.gitlab.com/ce/api/lint.html) to validate your local .gitlab-ci.yml.
      It can be installed as a git pre-commit hook, preventing commit (and, by extension, push) of an invalid .gitlab-ci.yml.
    '';
```
the repo description was a bit too long for `meta.description`, so here's adaptation of the first readme paragraph (+ corrected some grammatical mistakes)